content/_includes/embed.macros.njk

```diff
@@ -309,6 +315,21 @@ params:
 {%- endif -%}
 {%- endmacro %}
 
+{% macro srcset(
+  src,
+  width=none
+) -%}
+  {%- set width = width or 640 -%}
```

Comment:
@mirisuzanne Any reason to do this vs just setting `width=640` above?

Reply:
I'm not actually sure… Do defaults work if the calling location passes `none` as the value?

Reply:
Yes, the default value is used in place of `none`.

Reply:
It looks like you changed this, and changed it back?

Reply:
Yeah, it works with `none`, but apparently not with `""`. 🤷